src/content_script.js

```diff
@@ -40,6 +40,9 @@ const LOGIN_PATTERN_DETECTION_SELECTORS = [
   "[aria-label*='Log in with Facebook']", // Tinder
   "[action*='facebook_login']", // Airbnb
   "[action*='facebook_signup']", // Airbnb
+  "[data-tag='login-form'] ~ span button[color='facebookBlue']", // Patreon
+  ".welcome-buttons .btn-facebook", // Mixcloud homepage
+  ".auth-form .btn-facebook", // Mixcloud login modal
```

Comment:
Can you also add `.modal-content` to this selector? This still feels still broad. I also recommend revising the `.btn-facebook` to include the div: `div.btn-facebook`